src/txdb.cpp

```diff
@@ -328,23 +328,23 @@ bool CBlockTreeDB::ReadAddressUnspentIndex(uint160 addressHash, int type, CAddre
     return true;
 }
 
-bool CBlockTreeDB::WriteAddressIndex(const std::vector<CAddressAmount> &vect) {
+bool CBlockTreeDB::WriteAddressIndex(const CAddressIndexAmountVec &vect) {
```

Comment:
No functional change; I had added some `typedef`s in the previous commit but didn't update all the places that could use the new typedef (here's one of them).